packages/atomic/src/components/search/atomic-search-box/atomic-search-box.tsx

```diff
@@ -590,30 +587,65 @@ export class AtomicSearchBox {
     if (index === lastIndex && item.hideIfLast) {
       return null;
     }
+
+    const isButton = item.onSelect || item.query;
+    const commonAttributes = {
+      id: id,
+      key: item.key,
+      part: this.makeSuggestionPart(isSelected, !!item.query, item.part),
+      class: `flex px-4 min-h-[40px] items-center text-neutral-dark hover:bg-neutral-light cursor-pointer ${
+        isSelected ? 'bg-neutral-light' : ''
+      }`,
+      ref: (el: HTMLElement | undefined) => {
+        if (isHTMLElement(item.content)) {
+          el?.replaceChildren(item.content);
+        }
+        el?.setAttribute(
+          'aria-label',
+          this.bindings.i18n.t(
```

Comment:
Can we extract the aria label content up top?

Reply:
no because we may need to read the rendered content